toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -394,3 +402,70 @@ func addCustomizerRelease(imageChroot *safechroot.Chroot, toolVersion string, bu
 
 	return nil
 }
+
+func handleSELinux(selinuxMode imagecustomizerapi.SELinux, selinuxDisablePackageCheck bool,
+	partitionsCustomized bool, imageChroot *safechroot.Chroot,
+) error {
+	var err error
+
+	// Resolve the default SELinux mode.
+	if selinuxMode == imagecustomizerapi.SELinuxDefault {
+		selinuxMode, err = getCurrentSELinuxMode(imageChroot)
+		if err != nil {
+			return err
+		}
+	}
+
+	// If the partitions were customized, then the grub.cfg file will have been recreated from scratch and therefore
+	// the SELinux args will already be correct and don't need to be updated.
+	if !partitionsCustomized {
+		// Update the SELinux kernel command-line args.
+		err := updateSELinuxCommandLine(selinuxMode, imageChroot)
+		if err != nil {
+			return fmt.Errorf("failed to update SELinux args in grub.cfg:\n%w", err)
+		}
+	}
+
+	if selinuxMode != imagecustomizerapi.SELinuxDisabled {
+		imagerSELinuxMode, err := selinuxModeToImager(selinuxMode)
+		if err != nil {
+			return err
+		}
+
+		if !selinuxDisablePackageCheck {
```

Comment:
wouldnt we still want to check that the user has installed some selinux policy package (when it's custom)?

maybe we could have 
SELinuxMode:
SELinuxPolicy: (when not specified, assume default) 
-- pretty similar to the original implementation

I guess the feedback / question earlier was that the policy could be installed based on the above fields... but as you mentioned PackagesInstall allows the user to specify the version they want which makes sense to me. So the previous implementation was ok / better imo

Reply:
The check if really just a sanity check for the user. If they are using their own custom SELinux policy, then we'll trust that they know what they are doing and get out of their way.

Checking that the policy package that the user specified is installed seems kind of pointless. If the user knows that package should be installed, then can just make sure it is installed.